app/assets/stylesheets/components/page_specific/_stripe_banner.scss

```diff
@@ -0,0 +1,70 @@
+// Stripe banner which appears on the homepage
+//
+// Styleguide Stripe banner
+
+.stripe-banner {
+  background: #f3f3f3 url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI4IiBoZWlnaHQ9IjgiIHZpZXdCb3g9IjAgMCA4MCA4MCIgZW5hYmxlLWJhY2tncm91bmQ9Im5ldyAwIDAgODAgODAiPjxwYXRoIHN0cm9rZT0iIzAwMCIgZD0iTTAgODBMODAgMCIvPjwvc3ZnPg==);
```

Comment:
I tried to do that before but ran into issues in IE which are highlighted here:

http://css-tricks.com/probably-dont-base64-svg/#comment-1586218

If you end up URL encoding the whole thing, as suggested then it's possibly going to end up being even bigger than the base64 version